### PR TITLE
Add two examples for Path::new

### PR DESCRIPTION
**Pull request by nham**

### CODE REVIEW

**Comment by steveklabnik** [comment]
Thanks for this! A few nits inline:

File: src/libstd/path.rs [hunk 1]
```diff
@@ -1248,6 +1248,13 @@ impl Path {
     /// use std::path::Path;
     ///
     /// Path::new("foo.txt");
+    ///
+    /// // Strings work too
```
Comment (steveklabnik) [inline src/libstd/path.rs]: There should be a or colon at the end of this sentence

File: src/libstd/path.rs [hunk 2]
```diff
@@ -1248,6 +1248,13 @@ impl Path {
...
+    /// let s = String::from("bar.txt");
```
Comment (steveklabnik) [inline src/libstd/path.rs]: There should be triple backticks: ` on the line before the example starts

File: src/libstd/path.rs [hunk 3]
```diff
@@ -1248,6 +1248,13 @@ impl Path {
...
+    /// let p = Path::new(&s);
+    ///
```
Comment (steveklabnik) [inline src/libstd/path.rs]: and after too

File: src/libstd/path.rs [hunk 4]
```diff
@@ -1248,6 +1248,13 @@ impl Path {
...
+    /// // As do other Paths
```
Comment (steveklabnik) [inline src/libstd/path.rs]: `Path`s should have the `s around it, and a period or a colon, as well as the newlines and triple backticks

**Comment by steveklabnik** [comment]
Just some nits! see http://doc.rust-lang.org/nightly/book/documentation.html for some examples of the style

**Comment by nham** [comment]
Sorry, I'm a bit confused on how to add the triple backticks. If these get split into multiple code blocks, will I have to import `std::path::Path` for each?

**Comment by steveklabnik** [comment]
@nham sorry i missed this notification! This looks great though, thanks.

(and yes, you'd have to import each time, they're totally separate examples.

**Pull request merged**